machine_management/configuring-hpa-for-an-application.adoc

```diff
@@ -0,0 +1,25 @@
+[id='configuring-hpa-for-an-application']
+= Configuring HPA for an application
+include::modules/common-attributes.adoc[]
+:context: application-hpa
+
+toc::[]
+
+You can configure Horizontal Pod Autoscaling (HPA) for an application that exports metrics.
+
+:FeatureName: Horizontal Pod Autoscaling for an application
+include::modules/technology-preview.adoc[leveloffset=+0]
+
+.Prerequisites
+
+* Log in as a user with administrative privileges for the cluster.
+* Make sure you configured the cluster for application monitoring. In the following example, it is presumed that Prometheus and Alertmanager instances were installed in the `default` namespace. See xref:../monitoring/application-monitoring.adoc#application-monitoring[Application monitoring].
+* Make sure you configured monitoring for your application. In the following example, it is presumed that the application and the service monitor for it were installed in the `default` namespace. See xref:../monitoring/application-monitoring.adoc#application-monitoring[Application monitoring].
+* Make sure your application exposes metrics using the `prometheus-adapter`. In the following example, it is presumed that `prometheus-adapater` was installed in the `default` namespace. See xref:../monitoring/application-monitoring.adoc#application-monitoring[Application monitoring].
```

Comment:
> Make sure that your application exposes metrics using the `prometheus-adapter`

Add a link to the right section of the docs to expose the metrics. Consider including the modules instead of the prereq link.